exercises/crypto-square/README.md

```diff
@@ -47,9 +47,10 @@ imtgdvsfearwermayoogoanouuiontnnlvtwttddesaohghnsseoau
 
 Output the encoded text in chunks.  Phrases that fill perfect rectangles
 `(r X c)` should be output `c` chunks of `r` length, separated by spaces.
-Phrases that do not fill perfect rectangles will have `n` empty spaces.
-Those spaces should be distributed evenly, added to the end of the last
-`n` chunks.
+Phrases that do not fill perfect rectangles will be `n` characters short.
+To minimize the jaggedness, the missing chars should be distributed evenly,
+added to the end of the last `n` chunks. The bottom row(s) should be, at most,
```

Comment:
I understand the intent, though the meaning of adding a missing character might imply that the bottom row gets longer. I'd phrase it either in terms of taking a character away or purely in terms of the bottom rows being shorter (which this PR is doing)

Reply:
Turns out the Rust implementation of the exercism spec got this requirement wrong.  See my comment, above.